src/Symfony/Component/PropertyAccess/PropertyAccess.php

```diff
@@ -19,39 +21,30 @@
 final class PropertyAccess
 {
     /**
-     * Creates a property accessor with the default configuration.
+     * Creates a property accessor.
+     *
+     * For dealing with several property accessor configured differently, use
+     * the createPropertyAccessorBuilder() method instead.
      *
-     * @param bool $throwExceptionOnInvalidIndex
-     * @param bool $magicCall
+     * @param bool                        $throwExceptionOnInvalidIndex
+     * @param bool                        $magicCall
+     * @param CacheItemPoolInterface|null $cacheItemPool
      *
      * @return PropertyAccessor The new property accessor
      */
-    public static function createPropertyAccessor($throwExceptionOnInvalidIndex = false, $magicCall = false)
+    public static function createPropertyAccessor($magicCall = false, $throwExceptionOnInvalidIndex = false, CacheItemPoolInterface $cacheItemPool = null)
```

Comment:
those properties should not exist as well because that is where the builder should be used. otherwise this static method is just a wrapper around the constructor of PropertyAccessor.